metadata-integration/java/datahub-event/src/main/resources/MetadataChangeProposal.avsc

```diff
@@ -99,7 +99,7 @@
     "type" : {
       "type" : "enum",
       "name" : "ChangeType",
-      "namespace" : "com.linkedin.pegasus2avro.events.metadata",
```

Comment:
Hmm this seems unrelated but was applied when i ran `./gradlew :metadata-service:openapi-servlet:spotlessApply`